pkg/sfu/rtpmunger.go

```diff
@@ -72,9 +72,12 @@ type RTPMunger struct {
 
 	extLastSN       uint64
 	extSecondLastSN uint64
-	extLastTS       uint64
-	tsOffset        uint64
-	lastMarker      bool
+	snOffset        uint64
```

Comment:
caching the latest offset and using it. `RangeMap.GetValue` was poking its head in the profile.